server.js

```diff
@@ -0,0 +1,77 @@
+const express = require("express");
+const server = express();
+const PORT = 7777;
+const path = require("path");
+const hbs = require("hbs");
+// require("dotenv").config();
+
+// / Make everything inside of public/ available to the browser (styles, images, frontend scripts)
+server.use(express.static(path.join(__dirname, "public"))); // rock solid syntax
+// indicates express where our "views" templates are located
+// view is a term part of MVC design pattern... more coming. in a nutshell, a view is a page ; )
+server.set("views", path.join(__dirname, "views"));
+// indicates express wich view engine this app will use ; )
+server.set("view engine", "hbs"); // in this case hbs, there are others ... pug/jade etc.engine !
+// server.use(express.urlencoded({extended:false}));
+hbs.registerPartials(path.join(__dirname, "views/partials"));
+// what is a view engine ???
+// basicaly a way to code HTML++ (loops, conditionals
+hbs.registerPartials(path.join(__dirname, "views/partials"));
+hbs.registerPartial("navBar", "{{navBar}}");
+hbs.registerPartial("footer", "{{footer}}");
```

Comment:
You already registered the view/partials folder, so any partial you put inside will be known by hbs, so you don't have to register them again manually : )